cyphermain.py

```diff
@@ -132,7 +147,8 @@ def select_files():
         for file in files:
             if file.endswith(tuple(ext)):
                 files_to_enc.append(os.path.join(root, file))
-
+    
+    encrypted_count = str(len(files_to_enc)) ## Get a count of the encrypted files

```

Comment:
you can't `str(len(array)` str sets the type to string, you are looking at an array here `files_to_enc` . This will throw a type error.